duckduckgo/macos-browser
# sync show or enter code

Task/Issue URL: https://app.asana.com/0/0/1204445114333864/f
Tech Design URL:
CC:

**Description**:

Presents the show or enter code view.

Some of the UX here is not great, but Tim is going to think about that separately.

**Steps to test this PR**:
1. Turn on sync on macOS
4. On macOS tap "Show or Enter Code"
5. Scan the QR code on another device (should be a recovery code)
6. Paste in a code - should only allow connect code.  Other device should connect.



---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
